float64 angle
float64 angle_dot
float64 velocity
float64 u_1
---
float64 u_1
float64 percent_safe
float64 cost
